derating/space-derating.stanza

```diff
@@ -5,31 +5,31 @@ defpackage ocdb/space-derating :
 public var DESIGN-TEMPERATURE:Double = 0.0
 
 ; RESISTOR DERATING PARAMETERS
-public val DERATE-RESISTOR-POWER = 0.6
-public val DERATE-INDUCTOR-POWER = 0.6
+public var DERATE-RESISTOR-POWER = 0.6
+public var DERATE-INDUCTOR-POWER = 0.6
```

Comment:
SIdenote: these are now `var` in case a user wishes to customize deratings in a design.